Sources/tart/URL+Prunable.swift

```diff
@@ -10,10 +11,30 @@ extension URL: Prunable {
   }
 
   func allocatedSizeBytes() throws -> Int {
-    try resourceValues(forKeys: [.totalFileAllocatedSizeKey]).totalFileAllocatedSize!
+    let values = try resourceValues(forKeys: [.totalFileAllocatedSizeKey, .mayShareFileContentKey])
+    // make sure the file's origin file is there and duplication works
+    var dedublicatedSize = 0
+    if values.mayShareFileContent == true {
+      dedublicatedSize = Int(deduplicatedBytes())
+    }
+    return values.totalFileAllocatedSize! - dedublicatedSize
   }
 
   func sizeBytes() throws -> Int {
     try resourceValues(forKeys: [.totalFileSizeKey]).totalFileSize!
   }
+
+  func setDeduplicatedBytes(_ size: UInt64) {
+    var intVal = size
+    let data = Data(bytes: &intVal, count: MemoryLayout.size(ofValue: intVal))
```

Comment:
What do you think about storing the values as strings instead?

This has two benefits:

* no need to bother with endianness
* `xattr -l` will be able to show the value that we set, which aids in debugging

Reply:
SGTM. See [a42174c](https://github.com/cirruslabs/tart/pull/870/commits/a42174c780ec8a468ae805700c4b66ad3bc0a47c)